common/gamelib/move-check.mjs

```diff
@@ -15,6 +15,7 @@ const PLAYER_MOVE_DISTANCE_LIMIT = 1; // cell
 function checkPlayerMove(oldPlayerData, updateMessage, gameMap, clientSide=false) {
   let canMove = true;
   canMove = canMove && _speedCheck(oldPlayerData, updateMessage, clientSide);
+  if (!canMove) console.debug('- speed check failed');
```

Comment:
It seems that we can't easily disable this log message in production, I think we should have something to allow filtering the log level.